src/components/responsive-flyout/styles/CloseButton.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../../styles/variables';
+@import './variables';
+
+.btn.bdl-close-btn {
+    width: $close-btn-size;
+    height: $close-btn-size;
+    padding: unset;
```

Comment:
Should we be using `PlainButton`, instead?

Reply:
Button currently has the styling I need. However, styling might change in the next week or so according to design. Perhaps I can make the switch the: use PlainButton and provide explicit CSS styling